talk/owt/sdk/include/cpp/owt/base/globalconfiguration.h

```diff
@@ -88,6 +88,31 @@ class GlobalConfiguration {
     hardware_acceleration_enabled_ = enabled;
   }
 #endif
+
+#if defined(WEBRTC_WIN)
+  /**
+   @brief Enable driver-based super resolution for video rendering if underlying
+   platform supports it. This can only enabled on Windows platform and is by default
+   turned off.
+   Be noted turning this on does not neccessarily enable super resolution on all hardwares.
```

Comment:
Please describe the hardware and software(driver) requirements for SR support.

Reply:
done.